public/app/features/plugins/loader/systemjsHooks.ts

```diff
@@ -85,3 +84,24 @@ function getBackWardsCompatibleUrl(url: string) {
 
   return hasValidFileExtension ? url : url + '.js';
 }
+
+// This function takes the path used in loadPluginCss and attempts to resolve it
+// by checking the SystemJS entries for a matching pluginId then using that entry to find the baseUrl.
+// If no match is found then it returns a fallback attempt at a relative path.
+export function getLoadPluginCssUrl(path: string) {
```

Comment:
should `path` be called `id`? I got  a bit confused with the previous function accepting an id and this one accepting a path.

Reply:
Yeah it's easier to scan it we keep it as `id`. I've renamed it.  👍